Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM alpine:3.15.0
```

Comment:
Do we want to remove the point to allow patches at any job trigger?

Reply:
_maybe_, the base image is to ensure as many layers are shared between services as possible, and to ensure that we know exactly what is running, we can control and update this periodically though.